src/Raven.Client/Replication/Messages/ReplicationBatchReply.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Raven.Abstractions.Replication
+{
+    public class ReplicationBatchReply
+    {
+		public enum ReplyType
+		{
+			Success,
```

Comment:
Add None as the first option here. 
We want to make sure that if derserialization fails, we have a nonsense value here.